src/components/views/rooms/BasicMessageComposer.tsx

```diff
@@ -567,7 +577,7 @@ export default class BasicMessageEditor extends React.Component<IProps, IState>
 
     private onAutoCompleteSelectionChange = (completion: ICompletion, completionIndex: number) => {
         this.modifiedFlag = true;
-        this.props.model.autoComplete.onComponentSelectionChange(completion);
+        // this.props.model.autoComplete.onComponentSelectionChange(completion);
```

Comment:
but whyyyy....

(add comment or remove please)

Reply:
Oooh this pointed out more code I can 🗑️